spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/data/redis/RedisAutoConfigurationTests.java

```diff
@@ -167,13 +172,34 @@ void testRedisConfigurationWithPool() {
 				});
 	}
 
-	@Test
-	void testRedisConfigurationWithTimeout() {
-		this.contextRunner.withPropertyValues("spring.redis.host:foo", "spring.redis.timeout:100").run((context) -> {
-			LettuceConnectionFactory cf = context.getBean(LettuceConnectionFactory.class);
-			assertThat(cf.getHostName()).isEqualTo("foo");
-			assertThat(cf.getTimeout()).isEqualTo(100);
-		});
+	@ParameterizedTest(name = "{0}")
+	@MethodSource("timeoutArguments")
+	void testRedisConfigurationWithTimeouts(List<String> properties, long readTimeout, long connectionTimeout) {
```

Comment:
Same, I don't think a parameterized test is warranted here.

Reply:
done